rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -0,0 +1,896 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hsearch" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis for migration from Hibernate Search 4.x to Hibernate Search 5.x.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.4.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
+        </dependencies>
+        <sourceTechnology id="hibernate-search" versionRange="[4,)" />
+        <targetTechnology id="hibernate-search5" versionRange="[5,)" />
+        <targetTechnology id="eap7" versionRange="[7,)" />
+        <tag>reviewed-2016-04-12</tag>
+    </metadata>
+    <rules>
+        <rule id="hsearch-00000">
+            <when>
+                <javaclass references="org.hibernate.search.{*}" />
+            </when>
+            <perform>
+                <classification title="Hibernate Search usage" effort="0" severity="mandatory">
```

Comment:
Absolutely, I know the difference (now; I didn't know it a couple of weeks ago, so I will probably have to file a JIRA for the existing classifications because I think that most of them, if not all, should in fact be hints). I think adding classifications like this is a _good_ idea, but the other rules we have for EAP 6 -> 7 don't do that, so I question the consistency.